sdk/publishing.md

```diff
@@ -1,9 +1,8 @@
 # Publishing Runbook
 
 When preparing to make a release of the SDK follow these steps:
-1. Update version numbers of all crates to be published from `0.0.X` to `0.0.X
-   + 1` including updating the version requirement for any packages which have
-   workspace dependencies which are also being published.
+1. Update version numbers of all crates to be published from `0.0.X` to `0.0.X + 1` including updating the version requirement for any packages which have
```

Comment:
When I look at this I wonder, if it is 0.0.3, does it go to 1.1.4 or 0.0.4 or ? what does that `+1` apply to? Major field, minor field or patch field? Can we show one or two examples to clarify this?

Reply:
it should be 0.0.{X + 1}, @jjleng

Reply:
Our rust SDK lags behind. We did not give much love to it. But in the future, we should also follow Semver. For now, I will approve to unblock. Please address Raj's comment.

Reply:
@rajkaramchedu - just added an example : )